webrtc-c/canary/src/CanaryWebrtc.cpp

```diff
@@ -182,7 +182,7 @@ VOID sendCustomFrames(Canary::PPeer pPeer, MEDIA_STREAM_TRACK_KIND kind, UINT64
     UINT32 actualFrameSize = 0;
     UINT32 frameSizeWithoutNalu = 0;
     // This is the actual frame size that includes the metadata and the actual frame data
-    actualFrameSize = CANARY_METADATA_SIZE + (dataRate / frameRate);
+    actualFrameSize = CANARY_METADATA_SIZE + ((dataRate / 8) / frameRate);
```

Comment:
The data rate is in bytes per second and frame rate is in frames per second. Why the divide by 8?

Reply:
Never mind. Did not notice the change below.